xlators/mount/fuse/utils/mount.glusterfs.in

```diff
@@ -302,8 +302,10 @@ start_glusterfs ()
         cmd_line=$(echo "$cmd_line --fuse-mountopts=$fuse_mountopts");
     fi
 
-    if [ -n "$xlator_option" ]; then
-        cmd_line=$(echo "$cmd_line --xlator-option=$xlator_option");
+    if [ -n "${xlator_options}" ]; then
```

Comment:
It would break existing behavior of the mount.glusterfs, we should not break existing behavior.

Reply:
Please clarify exactly what this would break then I'll sort that out.  Lower down where xlator_option is set, I'm now appending to the array instead, so if the array is not-empty it will loop in here.

The only thing I can think of that would be affected here is if someone pre-sets the environment, eg:

```
xlator_option="..." /sbin/mount.glusterfs
```

Or are you referring to the fact that you can now actually process multiple --xlator-option values at all?  So someone that previously passed multiple options would only have had the last one set but expecting to have them all set, so I consider this a fix of behaviour.

Reply:
I missed the condition to understand the -xlator-option ins mount.glusterfs.in